cylc/flow/cfgspec/workflow.py

```diff
@@ -416,6 +416,40 @@ def get_script_common_text(this: str, example: Optional[str] = None):
                The default time zone is now ``Z`` instead of the local time of
                the first workflow start.
         ''')
+        with Conf('experimental', desc='''
```

Comment:
Think we can have this as a global option too? That way we don't have to alter all our workflows twice (once to test, and once to undo when it goes live)